make/kube-dist

```diff
@@ -6,33 +6,31 @@ GIT_ROOT="${GIT_ROOT:-$(git rev-parse --show-toplevel)}"
 
 . "${GIT_ROOT}/make/include/versioning"
 
-artifacts=("")
-for variant in kube helm ; do
-    tmp_dir=$(mktemp -d)
-    trap "rm -rf '${tmp_dir}'" EXIT
-    artifact="output/${ARTIFACT_NAME}-${variant}-${APP_VERSION}.tgz"
-
-    rm -f "${GIT_ROOT}/${artifact}"
-
-    # Add "-opensuse" suffix to the chart directory if we are on opensuse stemcell.
-    # The directory should match the chart name.
-    if [[ $FISSILE_STEMCELL =~ ^.*fissile-stemcell-opensuse.*$ ]] ; then
-      suffix="-opensuse"
-    fi
+if [[ ${FISSILE_STEMCELL} =~ ^.*fissile-stemcell-opensuse.*$ ]] ; then
+    stemcell_os="opensuse"
+elif [[ ${FISSILE_STEMCELL} =~ ^.*fissile-stemcell-sle.*$ ]] ; then
+    stemcell_os="sle"
+else
+    echo "Unkown stemcell operating system: ${FISSILE_STEMCELL}"
```

Comment:
Typo `Unkown` vs `Unk>n<own`.

Reply:
Thanks! Fixed in 00f35b008877c3f9bdc9bd79db510aab33d7833e along with `make/kube-dist` where I copied this code from